scripts/start

```diff
@@ -5,19 +5,26 @@ cd $(dirname $0)/..
 exists() {
     [ -e "$1" ]
 }
+if [[ $(uname -m) != "arm"* ]]; then
+    PROFILE=--profile x86
+else
+    echo "some modules are not ARM compatible yet so skipping them."
+    PROFILE=
+fi
 
 if exists Artemis-modules-extra; then
     ADDITIONAL_OPTIONS=--scale=karton-sqlmap=5
 else
+    echo "you do not have Artemis-modules-extra installed. please refer to https://artemis-scanner.readthedocs.io/en/latest/quick-start.html"
```

Comment:
use the language of benefits - why should anyone want to install these modules? what do they bring?

tl;dr: CERT pl finds tens of thousands of misconfigured SSL and hundreds of SQL injections using these modules ;)

Reply:
I think we may even show what command to run (just tell that the modules have different licenses) so that the burden on the user is as low as possible, they just need to copy a comamnd